cpp/tools/parquet/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ if(PARQUET_BUILD_EXECUTABLES)
     install(TARGETS ${TOOL} ${INSTALL_IS_OPTIONAL}
             RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
   endforeach(TOOL)
-  target_link_libraries(parquet-dump-footer ${ARROW_LIBRARIES})
+  # target_link_libraries(parquet-dump-footer ${ARROW_LIBRARIES})
```

Comment:
Could you remove this entirely instead of commenting out?

Reply:
Done.